security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/ConnectivityResult.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.results;
+
+import java.util.Set;
+
+/**
+ * @author Bertrand Rix <bertrand.rix at artelys.com>
+ */
+public class ConnectivityResult {
+
+    private final int createdSynchronousComponentCount;
+
+    private final int createdConnectedComponentCount;
+
+    private final double loadActivePowerLoss;
+
+    private final double generationActivePowerLoss;
+
+    private final Set<String> lostElements;
+
+    public ConnectivityResult(int createdSynchronousComponentCount, int createdConnectedComponentCount, double loadActivePowerLoss,
```

Comment:
As discuss in TSC yesterday, it seems that the naming is not the good one. 
- `loadActivePowerLoss` can be understand as network losses. We suggest `disconnectedLoadActivePower`.
- `generationActivePowerLoss` : we suggest `disconnectedGenerationActivePower`.

Reply:
I renamed the two attributes.